### DISCUSSION TITLE
Are height maps supported?

### DISCUSSION THREAD

**Discussion by Flynsarmy** [spimort/TerraBrush#79] [Q&A] [answered]
I'd like to import heightmap data from world machine or some other application. Is that supported in this terrain editor? I see 'Height Map' has been moved to 'Deprecated (DO NOT USE)'. Is there an importer somewhere?

**Comment by spimort** [comment]
Hey @Flynsarmy !
As of now, there are no import "wizard" heightmap but you can definitely use yours if you have something.
The deprecated section is only there as a transition for the new zones feature.
Now all the heightmaps go under the "Zones" section : 
![image](https://github.com/spimort/TerraBrush/assets/6035485/ba3ab3ab-a094-4ef1-8c8a-b736ccde9d95)

The image generated by the plugin is a RFloat image (OpenGL texture format GL_R32F where there's one component, a 32-bit floating-point), so if you have something like that, you can just use it in TerraBrush : 
![image](https://github.com/spimort/TerraBrush/assets/6035485/d535c807-8010-45b1-b780-03491610ead7)

I never used world machine, I'll have to take a look at it.

**Comment by spimort** [comment]
I'll convert this issue to a discussion since this not really an issue but a question 🙂

**Comment by spimort** [comment, marked as answer]
@Flynsarmy Hey, I know this question was asked a long time ago, but it's possible now with the version 0.10.0Alpha 😎!
